### ISSUE TITLE
Add PHP 8 support please

### ISSUE THREAD

**Issue by anaggh** [tamtamchik/simple-flash#12]
**Describe the bug**
```
Warning: Private methods cannot be final as they are never overridden by other classes in /vendor/tamtamchik/simple-flash/src/Flash.php on line 31
```

**Comment by tamtamchik** [comment]
Thanks for the feedback! 
I'll try to make an update ASAP.

**Comment by tamtamchik** [comment]
Fixed in version: [2.0.1](https://github.com/tamtamchik/simple-flash/releases/tag/2.0.1)
